### PR TITLE
Prober not working

### PR DESCRIPTION
**Pull request by vincentgoat**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Important!!**
This PR should be merged after this fix https://github.com/kubeedge/kubernetes/pull/6 cherry-picked into tag (github.com/kubeedge/kubernetes v1.22.6-kubeedge3)

Prober such as liveness or readiness is not working when set in pod yaml.

**Which issue(s) this PR fixes**:https://github.com/kubeedge/kubeedge/issues/4453

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by vincentgoat** [comment]
@fisherxu @Shelley-BaoYue PTAL

File: go.mod [hunk 1]
```diff
@@ -108,7 +108,7 @@ replace (
 	k8s.io/kube-scheduler => github.com/kubeedge/kubernetes/staging/src/k8s.io/kube-scheduler v1.22.6-kubeedge3
 	k8s.io/kubectl => github.com/kubeedge/kubernetes/staging/src/k8s.io/kubectl v1.22.6-kubeedge3
 	k8s.io/kubelet => github.com/kubeedge/kubernetes/staging/src/k8s.io/kubelet v1.22.6-kubeedge3
-	k8s.io/kubernetes => github.com/kubeedge/kubernetes v1.22.6-kubeedge3
+	k8s.io/kubernetes => github.com/vincentgoat/ke-kubernetes v1.22.6-kubeedge-prober
```
Comment (zc2638) [inline go.mod:111]: Maybe commit the change of `github.com/vincentgoat/ke-kubernetes@v1.22.6-kubeedge-prober` to `github.com/kubeedge/kubernetes`, and then fix this PR

Comment (vincentgoat) [reply go.mod:111]: Yes, after this PR is reviewed.

**Comment by wackxu** [comment]
we can add some e2e to make sure the prober work correctly

File: edge/pkg/edged/edged.go [hunk 2]
```diff
@@ -273,13 +273,13 @@ func (e *edged) handlePod(op string, content []byte, podCfg *config.PodConfig) (
 		switch op {
 		case model.InsertOperation:
 			adds := &kubelettypes.PodUpdate{Op: kubelettypes.ADD, Pods: pods, Source: kubelettypes.ApiserverSource}
-			podCfg.EdgedCh <- *adds
+			podCfg.Channel(kubelettypes.ApiserverSource) <- *adds
```
Comment (wackxu) [inline edge/pkg/edged/edged.go:276]: we can get the  `podCfg.Channel(kubelettypes.ApiserverSource)` once and set it to the member of edged to reduce the call for `podCfg.Channel(kubelettypes.ApiserverSource)`

Comment (vincentgoat) [reply edge/pkg/edged/edged.go]: done

File: edge/pkg/edged/edged.go [hunk 3]
```diff
@@ -314,10 +314,10 @@ func (e *edged) handlePodListFromMetaManager(content []byte, podCfg *config.PodC
 	}
 
 	adds := &kubelettypes.PodUpdate{Op: kubelettypes.ADD, Pods: pods, Source: kubelettypes.ApiserverSource}
-	podCfg.EdgedCh <- *adds
+	podCfg.Channel(kubelettypes.ApiserverSource) <- *adds
 
 	updates := &kubelettypes.PodUpdate{Op: kubelettypes.UPDATE, Pods: podsUpdate, Source: kubelettypes.ApiserverSource}
-	podCfg.EdgedCh <- *updates
+	podCfg.Channel(kubelettypes.ApiserverSource) <- *updates
```
[the remark this answers is not in the document]
Comment (vincentgoat) [reply edge/pkg/edged/edged.go]: done

**Comment by vincentgoat** [comment]
Solved by https://github.com/kubeedge/kubeedge/pull/4509, close this PR.

**Pull request closed without merging**